Description: Fix building clang against a sysroot on armhf.
  This complements https://chromium.googlesource.com/chromium/src/+/930c527.
Author: Olivier Tilloy <olivier.tilloy@canonical.com>
Forwarded: not-yet

--- a/tools/clang/scripts/build.py
+++ b/tools/clang/scripts/build.py
@@ -796,6 +796,8 @@ def main():
     # Add the sysroot to base_cmake_args.
     if platform.machine() == 'aarch64':
       base_cmake_args.append('-DCMAKE_SYSROOT=' + sysroot_arm64)
+    elif platform.machine() == 'armv7l':
+      base_cmake_args.append('-DCMAKE_SYSROOT=' + sysroot_arm)
     else:
       # amd64 is the default toolchain.
       base_cmake_args.append('-DCMAKE_SYSROOT=' + sysroot_amd64)
